### PR TITLE
Search bar is able to submit even when no text has been imputed(Small screens)

### LINKED ISSUE #595: Search bar is able to submit even when no text has been imputed
**Describe the bug**
On the navbar, the search bar is currently doing a fetch request even if no data has been passed inside.

**To Reproduce**

1. Go to the Home page
3. Click on input field(NB: don't enter anything in the input field) in the navbar and hit enter or click on the search icon on the right side of the input
4. a request is being made
5. See the error

**Expected behavior**
In order to prevent useless requests, the search bar should only be able to submit when it has data imputed in.

**Screenshots**
image

### PR DESCRIPTION
**Pull request by yaya-mamoudou**
## Summary

Description of PR
This PR is in an addition to another merged [PR](https://github.com/unstructuredstudio/zubhub/pull/605).
After testing I realized that the user was still being able to submit an empty form, but this time when on mobile. 
I also observed an imparity of color balance in the input field when not being focused. 
Closes #595

## Changes

- Added the handleSubmit function to the second InputField
- Changed the colour of the input field to white when not focused and black when focused

## Screenshots
*Not focused*
image

*Focused*
image

### CODE REVIEW

**Comment by yaya-mamoudou** [comment]
@tuxology please review this PR when you can

**Comment by tuxology** [comment]
@yaya-mamoudou I planned to merge a dark search bar change being submitted at https://github.com/unstructuredstudio/zubhub/pull/584 Do you think the darker one looks better? Also I agree with the input field focused issue. Keep this PR open I'd say, we will merge #584, then you can improve this one?

**Comment by yaya-mamoudou** [comment]
Sure @tuxology. Let's do this that way.

**Comment by tuxology** [comment]
@yaya-mamoudou Can we update it since the #584 is merged now?

**Comment by yaya-mamoudou** [comment]
> @yaya-mamoudou Can we update it since the #584 is merged now?

@tuxology this PR was mainly to solve the issue with the search input on mobile submitting even when empty.
Should I do just that, or also include this style on the input when focused?

File: zubhub_frontend/zubhub/src/assets/js/styles/views/page_wrapper/pageWrapperStyles.js [hunk 1]
```diff
@@ -5,6 +5,20 @@ const styles = theme => ({
   input: {
     flex: 1,
     marginRight: '2rem',
+    '& .MuiInputBase-input': {
```
Comment (tuxology) [inline zubhub_frontend/zubhub/src/assets/js/styles/views/page_wrapper/pageWrapperStyles.js]: I think we can skip this. The current styling looks ok

**Review by tuxology — CHANGES_REQUESTED** [review]
We'll merge after we remove focus/color changes. We'll on do the fix for empty submission on mobile for now

**Comment by yaya-mamoudou** [comment]
@tuxology this is ready to merge

**Comment by tuxology** [comment]
Since too much has changed in top search bar, we will fix this subset of issue related to #605 later  - closing this

**Pull request closed without merging**